src/Config_Command.php

```diff
@@ -421,15 +421,50 @@ public function list_( $args, $assoc_args ) {
 	 * @when before_wp_load
 	 */
 	public function get( $args, $assoc_args ) {
-		$path                = $this->get_config_path( $assoc_args );
-		$wp_config_file_name = basename( $path );
-		list( $name )        = $args;
-		$type                = Utils\get_flag_value( $assoc_args, 'type' );
-
-		$value = $this->return_value( $name, $type, self::get_wp_config_vars( $path ), $wp_config_file_name );
+		$value = $this->get_value( $assoc_args, $args );
 		WP_CLI::print_value( $value, $assoc_args );
 	}
 
+	/**
+	 * Determines whether value of a specific constant or variable defined is true.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <name>
+	 * : Name of the wp-config.php constant or variable.
+	 *
+	 * [--type=<type>]
+	 * : Type of config value to retrieve. Defaults to 'all'.
+	 * ---
+	 * default: all
+	 * options:
+	 *   - constant
+	 *   - variable
+	 *   - all
+	 * ---
+	 *
+	 * [--config-file=<path>]
+	 * : Specify the file path to the config file to be read. Defaults to the root of the
+	 * WordPress installation and the filename "wp-config.php".
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Assert if MULTISITE is true
+	 *     $ wp config is-true MULTISITE
```

Comment:
Let's also use `echo $?` in this example to show what the value is.

Reply:
I don't follow here. The command doesn't actually echo anything, so I'm not sure whether or not having the example echo something would add clarity. I may be missing something though in the functioning of the `help` command.

Reply:
Added and pushed.